src/canisters/frontend/ic-certified-assets/src/state_machine.rs

```diff
@@ -36,12 +36,17 @@ pub struct AssetEncoding {
     pub sha256: [u8; 32],
 }
 
+#[derive(Default, Clone, Debug, CandidType, Deserialize)]
+pub struct AssetProperties {
+    pub max_age: Option<u64>,
+    pub headers: Option<HashMap<String, String>>,
+}
+
 #[derive(Default, Clone, Debug, CandidType, Deserialize)]
 pub struct Asset {
     pub content_type: String,
     pub encodings: HashMap<String, AssetEncoding>,
-    pub max_age: Option<u64>,
-    pub headers: Option<HashMap<String, String>>,
+    pub properties: AssetProperties,
```

Comment:
Asset is a member of StableState.  What's the plan for upgradability?

Reply:
I decided to make this PR only about pretty printing 

(also smuggling small update to include `aliasing` in `get_/set_asset_properties`)